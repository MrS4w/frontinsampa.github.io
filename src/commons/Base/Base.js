import React from 'react';
import PropTypes from 'prop-types';

import CssBaseline from '@material-ui/core/CssBaseline';

const Base = ({ children }) => (
  <main>
    <CssBaseline />

    { children }
  </main>
);

Base.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Base;
