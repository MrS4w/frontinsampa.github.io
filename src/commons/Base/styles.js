import resetCss from 'reset-jss';
import normalizeCss from 'normalize-jss';

const styles = ({ palette }) => ({
  ...resetCss,
  ...normalizeCss,
  base: {
    backgroundColor: palette.tar,
    color: palette.live,
  },
});

export default styles;
