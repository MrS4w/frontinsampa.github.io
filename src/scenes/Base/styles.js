import resetCss from 'reset-jss';

const styles = ({ palette }) => ({
  ...resetCss,
  base: {
    backgroundColor: palette.dark,
  },
});

export default styles;
