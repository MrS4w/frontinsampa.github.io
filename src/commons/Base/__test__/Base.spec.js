import React from 'react';
import { shallow } from 'enzyme';

import Base from '../Base';

describe('Commons: <Base />', () => {
  const wrapper = shallow(
    <Base>
      <div>
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
      </div>
    </Base>,
  );

  it('Create a snapshot.', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
