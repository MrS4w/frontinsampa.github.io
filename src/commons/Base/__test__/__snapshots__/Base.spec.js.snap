// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Commons: <Base /> Create a snapshot. 1`] = `
<main>
  <CssBaseline />
  <div>
    Lorem ipsum dolor sit amet consectetur adipisicing elit.
  </div>
</main>
`;
